src/containers/shared/utils.js

```diff
@@ -78,6 +78,54 @@ const FORMAT_PRICE_DEFAULT_OPTIONS = {
   padding: 0,
 }
 
+export const isLaterVersion = (source, target) => {
+  if (source === ' N/A ') return false
```

Comment:
why does N/A have extra spaces?

Reply:
(Also, if you're going to use this special spacing in multiple files, please make it a shared constant :) )

Reply:
I added it for sorting and spacing. With the new isLaterVersion sort, I can just remove those extra spaces now.